templates/sync-template.yaml

```diff
@@ -16,20 +16,20 @@ spec:
           - name: "THOTH_DOCUMENT_ID"
           - name: "THOTH_FORCE_SYNC"
         artifacts:
-          - name: "document"
-            path: "/mnt/component/document/{{inputs.parameters.THOTH_DOCUMENT_ID}}"
+          - name: outputdocument
+            path: "/mnt/output/{{inputs.parameters.THOTH_DOCUMENT_ID}}"
       retryStrategy:
         limit: 5
       container:
         name: "graph-sync"
-        image: "graph-sync-job"
+        image: "quay.io/thoth-station/graph-sync-job:v0.6.0-dev"
```

Comment:
why do we want to pull it from quay rather than an imagestreamtag?

Reply:
This was the actual image I was using yesterday because it could not pull `image: "graph-sync-job"`